exercise/1-Python-Pandas/1-a-Python.ipynb

```diff
@@ -68,20 +68,20 @@
    "source": [
     "## Basic Data Types\n",
     "\n",
-    "- Numbers: integer, float, complex\n",
-    "- String `\"I'm a string\"`\n",
+    "- Number: integer, float, complex\n",
+    "- String: `\"I'm a string\"`\n",
     "- Boolean: True, False\n",
     "- List: `[1, 2, 3]`\n",
     "- Tuple: `(a, b, c)`\n",
     "- Set: `{a, b, c}`\n",
-    "- Dictionaries: `{key1: value1, key2: value2}`\n"
+    "- Dictionary: `{key1: value1, key2: value2}`\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Numbers"
+    "### Number"
```

Comment:
Same here: Keep "Numbers" as you can have these three distinct types of numbers. Alternatively, you may call them "Numerical" if you want to stick with singular naming.